pages/common/git-archive.md

```diff
@@ -0,0 +1,24 @@
+# git archive
+
+> Create an archive of files from a named tree.
+> More information: <https://git-scm.com/docs/git-archive>.
+
+- Create a tar archive from the contents of the latest commit on the current branch:
+
+`git archive --verbose -o {{target.tar}} HEAD`
+
+- Create a tar archive from the contents of a specific directory:
+
+`git archive -o {{target.tar}} HEAD:{{directory}}`
+
+- Same as above, but change the prefix of all files in the archive:
```

Comment:
I'm not sure about this description, maybe the example is clearer if we say `preffix/` to include all the files inside a specific folder